src/components/network/network-table.js

```diff
@@ -144,31 +358,385 @@ const NetworkTable = (props) => {
+            if (
+                isLineOnEditMode(rowData) &&
+                rowData[columnDefinition.dataKey] !== undefined
+            ) {
+                const text = formatCell(rowData, columnDefinition);
+                const changeRequest = (value) =>
+                    registerChangeRequest(rowData, changeCmd, value);
+                const Editor = columnDefinition.editor;
+                if (Editor) {
+                    return (
+                        <Editor
+                            key={rowData.dataKey + rowData.id}
+                            className={clsx(
+                                classes.tableCell,
+                                classes.inlineEditionCell
+                            )}
+                            equipment={rowData}
+                            defaultValue={text}
+                            setter={(val) => changeRequest(val)}
+                            style={style}
+                        />
+                    );
+                }
+            }
+            return defaultCellRender(rowData, columnDefinition, key, style);
+        },
+        [
+            isLineOnEditMode,
+            defaultCellRender,
+            formatCell,
+            registerChangeRequest,
+            classes.tableCell,
+            classes.inlineEditionCell,
+        ]
+    );
+
+    function generateTableColumns(tabIndex) {
+        let generatedTableColumns = TABLES_DEFINITION_INDEXES.get(tabIndex)
+            .columns.filter((c) => {
+                return selectedColumnsNames.has(c.id);
+            })
+            .map((c) => {
+                let column = {
+                    ...c,
+                    label: intl.formatMessage({ id: c.id }),
+                    locked: lockedColumnsNames.has(c.id),
+                    columnWidth: c.columnWidth
+                        ? c.columnWidth
+                        : MIN_COLUMN_WIDTH,
+                };
+                if (c.changeCmd !== undefined) {
+                    column.cellRenderer = (
+                        rowData,
+                        columnDefinition,
+                        key,
+                        style
+                    ) =>
+                        editableCellRender(
+                            rowData,
+                            columnDefinition,
+                            c.changeCmd,
+                            key,
+                            style
+                        );
+                } else {
+                    column.cellRenderer = (
+                        rowData,
+                        columnDefinition,
+                        key,
+                        style
+                    ) =>
+                        defaultCellRender(
+                            rowData,
+                            columnDefinition,
+                            key,
+                            style
+                        );
+                }
+                delete column.changeCmd;
```

Comment:
Why remove column.changeCmd ? Not obvious to me.

Reply:
I reused some existing code to do this, but by removing it, the code is more complex.
So I changed it, stopped removing it, and now the code is simpler.